pypeit/core/wavecal/waveio.py

```diff
@@ -73,6 +73,9 @@ def load_template(arxiv_file, det, wvrng=None):
         binning of the template arc spectrum
 
     """
+    if not isinstance(arxiv_file, (str, pathlib.Path)) or arxiv_file is None or arxiv_file == "":
```

Comment:
Should we be also doing an `os.path.exists()` call here? Also note the inconsequential typo `non-existent`

Reply:
FWIW, I would recommend putting all of this error handling into `data.get_reid_arxiv_filepath` so that the first two lines of this function are:
```
calibfile, fmt = data.get_reid_arxiv_filepath(arxiv_file)
tbl = astropy.table.Table.read(calibfile, format=fmt)
```
As for this `if` statement, I think `isinstance` will catch the None case, so it can be
```
if not isinstance(arxiv_file, (str, pathlib.Path)) or arxiv_file == '':
```